e2e/client.e2e.test.ts

```diff
@@ -402,6 +402,22 @@ describe('createVariable', () => {
   });
 });
 
+describe('listTags', () => {
+  it('should lsit the tags', async () => {
+    const w = await client.Tags.list(organizationName);
+    const item = w.items[0];
+    expect(item).toHaveProperty('name')
+  });
+});
+
+describe('destroyTag',  () => {
```

Comment:
I am wondering: what happen if you send a delete request without any body ? Does it removes all the tags ? or an error ?
If it can delete all tags at once then it should be tested too ... but it is dangerous if it is ran in a real TFC organization...

Reply:
Also, I am not sure that the current implementation is able to remove multiple tags at once, which seems to be allowed at the API level.
We should add another test like:
```
it('should delete multiple tags', async () => {
        await client.Tags.delete('jwalab', [{type: "tags", id: "tag-xPfX5s5839Sjfghj"}, {type: "tags", id: "tag-xPfX5s5839Sjfghj"}]);
    });
```